server/app/controllers/meeting.js

```diff
@@ -121,7 +124,8 @@ router.post('/invite', function(req, res, next) {
         responseLink: 'http://ezpick.herokuapp.com/meeting/' + meeting.id,
       }
     );
-
+    newEmails = meeting.invited.concat(newEmails);
```

Comment:
Great call, I actually did something similar but it broke so I just swapped it into that. Don't know what I was thinking.